build.gradle

```diff
@@ -91,6 +91,7 @@ allprojects {
     hadoopVersion = '2.4.1'
     jettyVersion = '8.1.14.v20131031'
     gemfireXDVersion = '1.5.0-SNAPSHOT'
+    pulseVersion     = '1.4.1'
```

Comment:
Since the release is going to be 1.5.0, we should change the pulse version to be 1.5.0 here and in pulse repo.